schemas/EiffelTestCaseFinishedEvent/1.0.1.json

```diff
@@ -14,7 +14,7 @@
         },
         "version": {
           "type": "string",
-          "enum": [ "1.0.0" ],
+          "enum": [ "1.0.0", "1.0.1" ],
```

Comment:
You don't want to rename 1.0.0.json to 1.0.1.json. Instead, you want to copy it. 1.0.0.json should only check 1.0.0 of the event (i.e. `"enum": [ "1.0.0" ]`) and 1.0.1.json should only check 1.0.1 of the event (i.e. `"enum": [ "1.0.1" ]`). The former will not require data.outcome.metrics properties, but the latter will.